### PR TITLE
incorporating facts on the use of COBOL on today's era from OMP CWG's survey

### PR DESCRIPTION
**Pull request by ashiskumarnaik**
incorporating facts on the use of COBOL on today's era from OMP CWG's survey

## Proposed changes

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context to communicate to the maintainers why we should accept this pull request.

Fixes # (##263)

## Type of change

What type of changes does your PR introduce to the COBOL Programming Course? _Put an `x` in the boxes that apply._

- [ ] Bug fix (change which fixes one or more issues)
- [x] New feature (change which adds functionality or features to the course)
- [ ] Translations (change which adds or modifies translations of the course)
- [ ] Documentation (change which modifies documentation related to the course)
- [ ] This change requires an update to the course's z/OS environment

## Checklist:

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [Contributing Guideline](https://github.com/openmainframeproject/cobol-programming-course/blob/master/CONTRIBUTING.md)
- [x] I have included a title and description for this PR
- [x] I have DCO-signed all of my commits that are included in this PR
- [x] I have tested it manually and there are no regressions found
- [ ] I have commented my code, particularly in hard-to-understand areas (if appropriate)
- [ ] I have made corresponding changes to the documentation (if appropriate)

### CODE REVIEW

File: COBOL Programming Course #2 - Learning COBOL/COBOL Programming Course #2 - Learning COBOL.md [hunk 1]
```diff
@@ -42,21 +42,31 @@ Continuously updated to incorporate modernized and proven programming paradigms
 
 ## How is COBOL being used today?
 
-COBOL is everywhere. You have probably used an application written in COBOL today. For example, consider the following statistics:
+COBOL is omnipresent, and it's highly likely that you've interacted with a COBOL application today. Let's take a look at some compelling statistics:
 
-- About 95% of ATM swipes use COBOL code.
+- Approximately 95% of ATM transactions rely on COBOL code.
```
Comment (tanto259) [inline COBOL Programming Course #2 - Learning COBOL/COBOL Programming Course #2 - Learning COBOL.md:47]: ... rely on COBOL **codes**.

File: COBOL Programming Course #2 - Learning COBOL/COBOL Programming Course #2 - Learning COBOL.md [hunk 2]
```diff
@@ -42,21 +42,31 @@ Continuously updated to incorporate modernized and proven programming paradigms
...
-- Every day, COBOL systems handle $3 trillion in commerce.
+- On a daily basis, there are 200 times more COBOL transactions executed than there are Google searches.
+- Presently, there are over 250 billion lines of actively running COBOL programs, accounting for roughly 80% of the world's actively utilized code.
+- Every year, approximately 1.5 billion lines of new COBOL code are written.
 
-How pervasive is COBOL? Consider these mind-boggling facts:
+### Highlights
 
-- Every day there are 200 times more COBOL transactions executed than there are Google searches.
+Highlights from the survey conducted by OMP CWG shed further light on the use of COBOL in today's era:
```
Comment (tanto259) [inline COBOL Programming Course #2 - Learning COBOL/COBOL Programming Course #2 - Learning COBOL.md:59]: Can we expand **OMP CWG** to **Open Mainframe Project's COBOL Working Group** instead? Just to make it clear for the readers.

**Review by tanto259 — CHANGES_REQUESTED** [review]
Hi Ashis, thanks for your PR! Left some comments to fix some grammar stuff and also make it clear what OMP CWG is.

File: COBOL Programming Course #2 - Learning COBOL/COBOL Programming Course #2 - Learning COBOL.md (hunk 2 above)
Comment (ashiskumarnaik) [reply COBOL Programming Course #2 - Learning COBOL/COBOL Programming Course #2 - Learning COBOL.md]: @tanto259 I made the changes. you can review it now.

File: COBOL Programming Course #2 - Learning COBOL/COBOL Programming Course #2 - Learning COBOL.md (hunk 1 above)
Comment (ashiskumarnaik) [reply COBOL Programming Course #2 - Learning COBOL/COBOL Programming Course #2 - Learning COBOL.md]: @tanto259 I made the changes.

**Comment by ashiskumarnaik** [comment]
@tanto259 I have done the necessary changes . Please review it.

File: COBOL Programming Course #2 - Learning COBOL/COBOL Programming Course #2 - Learning COBOL.md [hunk 3]
```diff
@@ -42,21 +42,31 @@ Continuously updated to incorporate modernized and proven programming paradigms
...
-- Every day, COBOL systems handle $3 trillion in commerce.
+- On a daily basis, there are 200 times more COBOL transactions executed than there are Google searches.
+- Presently, there are over 250 billion lines of actively running COBOL programs, accounting for roughly 80% of the world's actively utilized code.
+- Every year, approximately 1.5 billion lines of new COBOL code are written.
 
-How pervasive is COBOL? Consider these mind-boggling facts:
+### Highlights from OMP CWG's survey on COBOL in 2021.
 
-- Every day there are 200 times more COBOL transactions executed than there are Google searches.
+Highlights from the survey conducted by OMP CWG (Open Mainframe Project's COBOL Working Group) shed further light on the use of COBOL in today's era:
```
Comment (tanto259) [inline COBOL Programming Course #2 - Learning COBOL/COBOL Programming Course #2 - Learning COBOL.md:59]: Could you remove the OMP CWG from all? I believe the Linux Foundation also have a Open Manufacturing Project and we don't want people to be confused.

So, something like "conducted by the Open Mainframe Project's COBOL Working Group shed further" would be great.

File: COBOL Programming Course #2 - Learning COBOL/COBOL Programming Course #2 - Learning COBOL.md [hunk 4]
```diff
@@ -42,21 +42,31 @@ Continuously updated to incorporate modernized and proven programming paradigms
...
-- COBOL powers 80% of in-person transactions.
+To truly grasp the extent of COBOL's prevalence, consider these mind-boggling facts:
 
-- Every day, COBOL systems handle $3 trillion in commerce.
+- On a daily basis, there are 200 times more COBOL transactions executed than there are Google searches.
+- Presently, there are over 250 billion lines of actively running COBOL programs, accounting for roughly 80% of the world's actively utilized code.
+- Every year, approximately 1.5 billion lines of new COBOL code are written.
 
-How pervasive is COBOL? Consider these mind-boggling facts:
+### Highlights from OMP CWG's survey on COBOL in 2021.
```
Comment (tanto259) [inline COBOL Programming Course #2 - Learning COBOL/COBOL Programming Course #2 - Learning COBOL.md:57]: Also this one

**Review by tanto259 — CHANGES_REQUESTED** [review]
Other than removing "OMP CWG" for possible confusion (especially since the COBOL Working Group was never shorten as CWG in publications), I have no other feedback. Thanks, Ashis!

**Review by tanto259 — APPROVED** [review]
Looks good to me, thanks, Ashis!

**Pull request merged**